ark/analysis/visualize.py

```diff
@@ -7,6 +7,57 @@
 from ark.utils import misc_utils
 
 
+def draw_barplot(data, x_col, y_col, x_label, y_label, title, figsize, color='#00FF00',
+                 dpi=None, save_dir=None, save_file=None):
+    """Draws a barplot for a given dataset
+
+    Args:
+        data (pandas.DataFrame):
+            Dataframe containing the two columns to visualize
+        x_col (str):
+            The name of the column to put on the x-axis
+        y_col (str):
+            The name of the column to put on the y-axis
+        x_label (str):
+            The name of the x label
+        y_label (str):
+            The name of the y label
+        title (str):
+            The name of the title to give the histogram
+            If None defaults to "Distribution of y vs x"
+        figsize (tuple):
+            A tuple determining the x and y dimension of the figure to plot
+        color (str):
+            The color of the bars to set
+        dpi (float):
+            The resolution of the image to save, ignored if save_dir is None
+        save_dir (str):
+            If specified, a directory where we will save the plot
+        save_file (str):
+            If save_dir specified, specify a file name you wish to save to.
+            Ignored if save_dir is None
+    """
+
+    # x_col must be valid
+    misc_utils.verify_in_list(x_col=x_col, column_names=data.columns.values)
+
+    # y_col must be valid
+    misc_utils.verify_in_list(y_col=y_col, column_names=data.columns.values)
+
+    # we need to set the plot to white since Seaborn likes to override this sometimes
+    fig = plt.figure(figsize=figsize)
+
+    # draw the barplot
+    sns.barplot(x=data[x_col], y=data[y_col], palette=[color for i in range(data.shape[0])])
```

Comment:
it might also be nice to allow a `kwargs` pass-down here in case people want to further customize their plots

Reply:
Yeah, I've been thinking about this too. For now, I think I'll leave this as is, but I'll open an issue for it since this applies to other functions in `visualize.py` as well.